flocker/common/script.py

```diff
@@ -118,19 +146,10 @@ def _parse_options(self, arguments):
 
     def main(self):
         """Parse arguments and run the script's main function via ``react``."""
-        observer = None
-        try:
-            if not self.log_directory.exists():
-                self.log_directory.makedirs()
-            log_path = self.log_directory.child(
-                b"%s-%d.log" % (os.path.basename(self.sys_module.argv[0]),
-                                os.getpid()))
-            log_file = log_path.open("a")
-            observer = FileLogObserver(log_file).emit
-            addObserver(observer)
-            msg("Arguments: %s" % (self.sys_module.argv,))
-        except (OSError, IOError):
-            pass
+        # XXX FLOC-936
+        # 0. If self.logging is False, don't do anything. Otherwise:
+        # 1. Create ThreadedLogWriter writing to stdout
```

Comment:
I wonder if there'd be any value (in general, most likely not for this issue) in having a non-threaded stdout logger using `twisted.internet.stdio`.